provisioning_templates/registration/linux_registration_default.erb

```diff
@@ -0,0 +1,27 @@
+<%#
+kind: registration
+name: Linux registration default
+model: ProvisioningTemplate
+oses:
+- CentOS
+- Fedora
```

Comment:
I think it now also works on Debian (and any platform that has curl and where puppetlabs_repo / puppet_setup works).

Reply:
Otherwise, looks good!

Reply:
I don't really understand the matching policy, so I can't say what OSes are available. I've rolled with what we have elsewhere :man_shrugging:
I guess we are missing some better oses targeting, but it's just for default selections, so I guess we're good with the most common ones.